### PR TITLE
Add scabbard state to database migrate subcommand

### PR DESCRIPTION
**Pull request by peterschwarz**

### CODE REVIEW

File: cli/src/action/database/postgres.rs [hunk 1]
```diff
@@ -55,6 +57,24 @@ pub fn postgres_migrations(url: &str) -> Result<(), CliError> {
         })?;
     }
 
+    #[cfg(feature = "scabbard-migrations")]
+    {
+        scabbard::migrations::run_postgres_migrations(conn!(pool)).map_err(|err| {
+            CliError::ActionError(format!(
+                "Unable to run Sqlite migrations for scabbard: {}",
```
Comment (agunde406) [inline cli/src/action/database/postgres.rs:64]: Sqlite -> Postgres

File: cli/src/action/database/postgres.rs [hunk 2]
```diff
@@ -55,6 +57,24 @@ pub fn postgres_migrations(url: &str) -> Result<(), CliError> {
...
+                err
+            ))
+        })?;
+
+        let postgres_backend: PostgresBackend = pool.into();
+        postgres_backend.run_migrations().map_err(|err| {
+            CliError::ActionError(format!(
+                "Unable to run Sqlite migrations for scabbard merkle state: {}",
```
Comment (agunde406) [inline cli/src/action/database/postgres.rs:72]: Sqlite -> Postgres

File: cli/Cargo.toml [hunk 3]
```diff
@@ -49,6 +49,17 @@ whoami = "0.7.0"
 path = "../libsplinter"
 features = ["admin-service", "registry"]
 
+[dependencies.scabbard]
```
Comment (agunde406) [inline cli/Cargo.toml:52]: typo in commit message: require -> required

**Review by agunde406 — CHANGES_REQUESTED** [review]

File: cli/Cargo.toml (hunk 3 above)
Comment (peterschwarz) [reply cli/Cargo.toml:52]: Fixed

File: cli/src/action/database/postgres.rs (hunk 1 above)
Comment (peterschwarz) [reply cli/src/action/database/postgres.rs]: Fixed

**Review by agunde406 — APPROVED** [review]

**Review by isabeltomb — APPROVED** [review]

**Pull request merged**